WoWPro/WoWPro_Frames.lua

```diff
@@ -452,7 +456,9 @@ function WoWPro.CustomizeFrames()
     for name, module in WoWPro:IterateModules() do
         if WoWPro[name].CustomizeFrames then WoWPro[name]:CustomizeFrames() end
     end
-    WoWPro.AnchorRestore(true) -- Just in case a module jiggled something
+    if not WoWPro.Minimized then
+        WoWPro.AnchorRestore(true) -- Just in case a module jiggled something
```

Comment:
Could you move the test for `WoWPro.Minimized` into `WoWPro.AnchorRestore` since I seem to infer that `WoWPro.AnchorRestore` should not do anything if `WoWPro.Minimized` is true.